### DISCUSSION TITLE
comfyui stuck on loading screen after latest update

### DISCUSSION THREAD

**Discussion by sandrosem** [comfyanonymous/ComfyUI#5219] [Q&A]
hey! I updated my comfyui to the latest version today and ever since restarting I'm stuck on the spinning wheel on the loading screen.... anyone has an idea what could have caused it and how to fix it?

**Comment by msseibel** [comment]
Seems to be related to the new frontend. Switch back to the old frontend as detailed on the bottom the readme...

--front-end-version Comfy-Org/ComfyUI_legacy_frontend@latest

**Comment by sandrosem** [comment]
thanks! I can now access the frontend again! however, now I have a new problem: comfy manager is gone and none of the nodes are recognised anymore.

**Reply by msseibel** [reply]
The manager is visible for me. 

I am stuck at installing `ComfyUI_IPAdapter_plus` but that is probably unrelated.

**Comment by marduk191** [comment]
this is 100% an issue and it also messes up the legacy interface. Effectively it makes it unusable. Just happened to me because i hit the stupid update all button.
